src/CommunityToolkit.Maui/Alerts/Snackbar/SnackBar.shared.cs

```diff
@@ -14,36 +14,46 @@ namespace CommunityToolkit.Maui.Alerts;
 /// <inheritdoc/>
 public partial class Snackbar : ISnackbar
 {
+	internal const string defaultActionButtonText = "OK";
```

Comment:
Maybe it should be in Defaults?

Reply:
Do we use `"OK"` as the default text in any other class?

I didn't add it to `Defaults` because I think it is only used in `Snackbar`.

Reply:
I think about Defaults as the way to have a global one time configuration for the app, so optional parameters can be skipped

Reply:
Added `Defaults.ActionButtonText` ✅